data/blog/12-week-cadence.md

```diff
@@ -0,0 +1,37 @@
+---
+title: New Electron Release Cadence
+author: sofianguy
+date: '2019-05-14'
+---
+
+🎉 Electron is moving to release a new stable version every 12 weeks! 🎉
+
+---
+
+## ⚡️ Wow that's quick! But why?
+
+Simply put, Chromium doesn't stop shipping so Electron is not going to slow down either.
+
+## 🚀 Why is 12 weeks the chosen cadence?
+
+Right now we are planning to upgrade every other Chromium version. Chromium ships a new stable version at a very quick pace of every 6 weeks. We have adjusted our stable release dates to match every other Chromium stable release. Up first, Electron v6.0.0 will include M76 and is scheduled for stable release on [July 30, 2019](https://electronjs.org/docs/tutorial/electron-timelines#600-release-schedule), the same release day as [Chromium M76](https://www.chromium.org/developers/calendar).
```

Comment:
We might want to link to M76 specifically here

Reply:
Do know a link for M76? I tried looking for a specific link but couldn't find any 🤔

Reply:
i'd say this might be better: https://www.chromestatus.com/features/schedule

Reply:
perfect! thank you